rules/CodeQuality/Rector/Class_/AddReturnDocBlockToCollectionPropertyGetterByToManyAttributeRector.php

```diff
@@ -110,7 +110,7 @@ public function refactorWithScope(Node $node, Scope $scope): ?Node
             $collectionObjectType = $this->getCollectionObjectTypeFromToManyAttribute($property);
 
             if (! $collectionObjectType instanceof FullyQualifiedObjectType) {
-                return null;
+                continue;
```

Comment:
`$hasChanged` flag before and after changed, to be verified after loop to return node or null is needed